CHANGELOG.md

```diff
@@ -0,0 +1,104 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- [Codelists] Blank Codelists may be created and given a name.
+- [Codelists] Codelists may be created from a string conforming to the Codelist Schema.
+- [Codelists] Codelists contain a set of Codes.
+- [Codelists] Codelists may be output as a XSD simpleType restriction containing enumeration elements.
```

Comment:
`output` -> `outputted`

Reply:
`outputted` is past tense - this is present.